.ci/lib.sh

```diff
@@ -56,7 +56,9 @@ function get_version(){
 	versions_file="$GOPATH/src/github.com/kata-containers/runtime/versions.yaml"
 	go get -d -u -v "$runtime_repo" || true
 	[ ! -f "$versions_file" ] && { echo >&2 "ERROR: cannot find $versions_file"; exit 1; }
-	yq read "$versions_file" "$dependency"
+	if [ "$ID" == "ubuntu" ] || [ "$ID" == "fedora" ]; then
```

Comment:
If you add this condition, centos will not be able to read the `version.yaml` file and we would not be able to get the versions we need.